src/server.tsx

```diff
@@ -35,7 +35,14 @@ app.options('*', cors());
 // Logging
 app.use((req: express.Request, res: express.Response, next: express.NextFunction) => {
     res.on('finish', () =>
-        console.log(JSON.stringify({ status: res.statusCode, method: req.method, path: req.path })),
+        console.log(
+            JSON.stringify({
+                status: res.statusCode,
+                method: req.method,
+                path: req.path,
+                didRenderEpic: res.locals.didRenderEpic,
```

Comment:
I wonder if there's a better way of structuring this. I _think_ nesting works. Perhaps overkill now, but maybe worth doing in future.